curl-sys/lib.rs

```diff
@@ -1,17 +1,13 @@
 #![allow(bad_style)]
 #![doc(html_root_url = "https://docs.rs/curl-sys/0.3")]
 
-extern crate libc;
+// These `extern crate` are required for conditional linkages of curl.
 #[cfg(link_libnghttp2)]
 extern crate libnghttp2_sys;
 #[cfg(link_libz)]
 extern crate libz_sys;
-#[cfg(feature = "mesalink")]
-extern crate mesalink; // ensure lib is linked to
```

Comment:
I think this needs to remain for the linkage too?

Reply:
Probably not? I thought it would be linked when feature "mesalink" is on. Anyway, updated at 5768b0b.